packages/use-optionally-controlled-state/src/index.tsx

```diff
@@ -1,23 +1,31 @@
-import {useState} from 'react';
+import {useState, useCallback, useRef} from 'react';
```

Comment:
Can you make sure `yarn lint` and `yarn test` passes? I didn't have time to set this up on CI yet.

The import of `useRef` should be flagged here, since this is not used.

Reply:
verified with yarn lint 👍